### PR TITLE
zathura: 0.5.2 -> 0.5.4

### PR DESCRIPTION
**Pull request by hmanhng**
## Description of changes
This version fixes flickering every time a file is changed, for example when modifying a latex file
[#258](https://git.pwmt.org/pwmt/zathura/-/issues/268)

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by donovanglover** [comment]
Result of `nixpkgs-review pr 278352` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages failed to build:</summary>
  <ul>
    <li>manga-cli</li>
    <li>vimPlugins.openscad-nvim</li>
    <li>zathura</li>
  </ul>
</details>

**Comment by donovanglover** [comment]
Build failure seems to be due to `xvfb-run`

```
error: builder for '/nix/store/qk1jw0aqvz735gzqpwhp02z23xjgcc41-zathura-0.5.4.drv' failed with exit code 1;
       last 10 log lines:
       > Program msginit found: YES (/nix/store/f0m95a8gfkwgijbiscbkn7f3ra6ha70x-gettext-0.21.1/bin/msginit)
       > Program msgmerge found: YES (/nix/store/f0m95a8gfkwgijbiscbkn7f3ra6ha70x-gettext-0.21.1/bin/msgmerge)
       > Program xgettext found: YES (/nix/store/f0m95a8gfkwgijbiscbkn7f3ra6ha70x-gettext-0.21.1/bin/xgettext)
       > Program sphinx-build found: YES (/nix/store/bk1hmcb7jh1xjrbqg76qswqswir2wclw-python3.11-sphinx-7.2.6/bin/sphinx-build)
       > Run-time dependency check found: YES 0.15.2
       > Program xvfb-run found: NO
       >
       > tests/meson.build:37:9: ERROR: Program 'xvfb-run' not found or not executable
```

Note that zathura currently fails to build on Darwin. See: https://github.com/NixOS/nixpkgs/issues/274276

**Comment by hmanhng** [comment]
> Build failure seems to be due to `xvfb-run`
> 
Sorry I forgot about this package, I'll recreate another pull request

**Comment by hmanhng** [comment]
@donovanglover 
There was something wrong with the `tests flag`, I built on a linux machine with the `disabled tests` flag and it was fine

**Comment by donovanglover** [comment]
Result of `nixpkgs-review pr 278352` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built:</summary>
  <ul>
    <li>manga-cli</li>
    <li>vimPlugins.openscad-nvim</li>
    <li>zathura</li>
  </ul>
</details>

File: pkgs/applications/misc/zathura/core/default.nix [hunk 1]
```diff
@@ -24,8 +24,9 @@ stdenv.mkDerivation (finalAttrs: {
     "-Dmanpages=enabled"
     "-Dconvert-icon=enabled"
     "-Dsynctex=enabled"
+    "-Dtests=disabled"
```
Comment (donovanglover) [inline pkgs/applications/misc/zathura/core/default.nix:27]: Could add a comment above explaining why tests are disabled

**Review by donovanglover — COMMENTED** [review]
Would change `tweak(zathura):` in the commit message to `zathura:` so it's easier to search for.

Besides that, it seems to work for me. Thanks for contributing and welcome to nixpkgs!

File: pkgs/applications/misc/zathura/core/default.nix (hunk 1 above)
Comment (hmanhng) [reply pkgs/applications/misc/zathura/core/default.nix:27]: I don't know what the problem is with libseccomp when testing
 could be here [#324](https://git.pwmt.org/pwmt/zathura/-/issues/324)

**Comment by stepbrobd** [comment]
The fix for darwin build is here https://github.com/NixOS/nixpkgs/pull/275534

**Comment by hmanhng** [comment]
> The fix for darwin build is here #275534

It has `tests` disabled by default for `darwin` [here](https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/misc/zathura/core/default.nix#L28), this is zathura's `tests` error with linux,maybe disable it temporarily.

**Comment by stepbrobd** [comment]
Result of `nixpkgs-review pr 278352` run on aarch64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>vimPlugins.openscad-nvim</li>
    <li>zathura</li>
  </ul>
</details>

**Pull request merged**